src/components/TransactionCompleted/index.tsx

```diff
@@ -21,10 +23,11 @@ const TransactionCompleted = ({
   isShowButtton,
   onButtonClick,
   buttonText,
+  rootStyle,
 }: TransactionCompletedProps) => {
   const theme = useContext(ThemeContext);
   return (
-    <Root>
+    <Root {...rootStyle}>
```

Comment:
what if `rootStyle` will be undefined ? I think it will break `...rootStyle`. You might want to add `...(rootStyle || {})` or set default value to `{}` at 26th line